src/main/java/cinemas/models/BookingRefund.java

```diff
@@ -0,0 +1,62 @@
+package cinemas.models;
+
+import cinemas.converters.ZonedDateTimeConverter;
+import jakarta.persistence.*;
+
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "booking_refunds")
+public class BookingRefund {
+    @Id
```

Comment:
Khóa chính là field id integer đi em ơi

Reply:
dạ em đổi toàn bộ khóa chính của các bảng sang integer rồi ạ